src/AssetPath.elm

```diff
@@ -1,18 +1,21 @@
-module AssetPath exposing (Asset(..), url)
+module AssetPath exposing (Asset(..), src, url)
 
 {-| The core API of `AssetPath`: a union type and a function.
 
-@docs Asset, url
+@docs Asset, url, src
+
 -}
 
-import String
+import Html.Attributes
```

Comment:
Build error says:

```
Cannot find type `Html.Attribute`.
55| src : Asset -> Html.Attribute msg
                   ^^^^^^^^^^^^^^
No module called `Html` has been imported. 
```
https://travis-ci.org/NoRedInk/elm-asset-path/builds/255841000

I think we want `import Html` in addition here. http://package.elm-lang.org/packages/elm-lang/html/2.0.0/Html#Attribute